Loading a functional Keras model throws NullPointerException.

#### Issue Description

- Expected behavior:
Keras model loads fine in Java/Scala using this library.
- Encountered behavior
Loading a Keras functional model with deeplearning4j is impossible, because a NullPointerException is thrown. This is the code used to load it (in Scala), and the thrown error:
```scala
object Model{
    val modelConfigPath = "../../model/model_config.json"
    val modelWeightsPath = "../../model/model_weights.h5"

    System.setProperty("org.bytedeco.javacpp.maxphysicalbytes", "4096M")
    System.setProperty("org.bytedeco.javacpp.maxbytes", "4096M")
    val model = KerasModelImport.importKerasModelAndWeights(modelConfigPath, modelWeightsPath)
    
}
````
The trace is:

> Caused by: java.lang.NullPointerException: null
>         at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelUtils.importWeights(KerasModelUtils.java:235)
>         at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:190)
>         at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:99)
>         at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelBuilder.buildModel(KerasModelBuilder.java:311)
>         at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasModelAndWeights(KerasModelImport.java:257)
>         at jobs.Model$.<clinit>(RotationGenerator.scala:28)


If I try to load the whole model (.h5 file with training data as well), I get slightly the same exception:

###### When I try to load without forcing configuration:
```scala
object Model{
    val modelPath = "../../model/best_image_v2_model.h5"
    val modelInputStream = new FileInputStream(modelPath)

    System.setProperty("org.bytedeco.javacpp.maxphysicalbytes", "4096M")
    System.setProperty("org.bytedeco.javacpp.maxbytes", "4096M")
    val model = KerasModelImport.importKerasModelAndWeights(modelInputStream, false)
}
```
I get the following error (same as above):

> Caused by: java.lang.NullPointerException: null
>         at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelUtils.importWeights(KerasModelUtils.java:235)
>         at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:190)
>         at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:99)
>         at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelBuilder.buildModel(KerasModelBuilder.java:311)
>         at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasModelAndWeights(KerasModelImport.java:150)
>         at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasModelAndWeights(KerasModelImport.java:50)
>         at jobs.Model$.<clinit>(RotationGenerator.scala:25)

###### When I try to load without the flag set:

```scala
object Model{
    val modelPath = "../../model/best_image_v2_model.h5"
    val modelInputStream = new FileInputStream(modelPath)

    System.setProperty("org.bytedeco.javacpp.maxphysicalbytes", "4096M")
    System.setProperty("org.bytedeco.javacpp.maxbytes", "4096M")
    val model = KerasModelImport.importKerasModelAndWeights(modelInputStream)
}
```

This is the error:
```scala
Caused by: java.lang.NullPointerException: null
        at org.deeplearning4j.nn.modelimport.keras.utils.KerasOptimizerUtils.mapOptimizer(KerasOptimizerUtils.java:116)
        at org.deeplearning4j.nn.modelimport.keras.KerasModel.importTrainingConfiguration(KerasModel.java:398)
        at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:174)
        at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:99)
        at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelBuilder.buildModel(KerasModelBuilder.java:311)
        at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasModelAndWeights(KerasModelImport.java:167)
        at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasModelAndWeights(KerasModelImport.java:68)
        at jobs.Model$.<clinit>(RotationGenerator.scala:25)
```

I have also tried to load this model using version 1.0.0-beta7, to get the following error:

> Caused by: org.deeplearning4j.nn.modelimport.keras.exceptions.InvalidKerasConfigurationException: Expected model class name Model (found Functional). For more information, see https://deeplearning4j.konduit.ai/keras-import/overview
>         at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:133)
>         at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:96)
>         at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelBuilder.buildModel(KerasModelBuilder.java:307)
>         at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasModelAndWeights(KerasModelImport.java:262)
>         at jobs.Model$.<clinit>(RotationGenerator.scala:25)
> 


This is the model structure as defined in python:
```python
def build_model_v2(model_image_width=64, model_image_height=128):
    """Loosely based on the U-net architecture"""
    encoder_input = keras.layers.Input(shape = ( model_image_height, model_image_width, 3,))
    conv_input = encoder_input
    conv_input = keras.layers.BatchNormalization()(conv_input)
    n_filters = [16, 16, 32, 32, 64, 64, 128, 128, 256, 256, 512, 512]
    prev_filter = n_filters[1]
    conv_layers = []
    for filter_size in n_filters:
        if filter_size != prev_filter:
            conv_input = keras.layers.MaxPooling2D((2, 2), padding='same')(conv_input)
        conv_input = keras.layers.Conv2D(filter_size, (3, 3), activation='relu', padding='same')(conv_input)
        conv_layers.append(conv_input)
        conv_input = keras.layers.BatchNormalization()(conv_input)
        prev_filter = filter_size
    conv_layers.reverse()
    conv_input = keras.layers.MaxPooling2D((2, 2), padding='same')(conv_input)

    def build_path(entry):
        nonlocal n_filters, conv_layers
        n_filters_rev = list(reversed(n_filters))
        prev_filter = n_filters_rev[0]
        for idx, filter in enumerate(n_filters_rev):
            if filter != prev_filter:
                entry = keras.layers.UpSampling2D((2, 2))(entry)
            entry = keras.layers.Conv2D(filter, (3, 3), activation='relu', padding='same')(entry)
            if idx > 1:
                entry = keras.layers.concatenate([entry, conv_layers[idx-2]])
            entry = keras.layers.BatchNormalization()(entry)
            prev_filter = filter
        entry = keras.layers.UpSampling2D((2, 2))(entry)
        entry = keras.layers.Conv2D(3, (3, 3), activation='sigmoid', padding='same')(entry)
        return entry

    left_rotation = build_path(conv_input)
    right_rotation = build_path(conv_input)
    back_rotation = build_path(conv_input)
    

    return keras.models.Model(encoder_input, [left_rotation, right_rotation, back_rotation])
```


#### Version Information

* Deeplearning4j version: Tried both 1.0.0-M1.1 and 1.0.0-M1, on 1.0.0-beta7 i get a different error - "Expected model class name Model (found Functional)"
* Platform information (OS, etc): Ubuntu 20.04 LTS

--- a/deeplearning4j/deeplearning4j-modelimport/src/main/java/org/deeplearning4j/nn/modelimport/keras/utils/KerasOptimizerUtils.java
+++ b/deeplearning4j/deeplearning4j-modelimport/src/main/java/org/deeplearning4j/nn/modelimport/keras/utils/KerasOptimizerUtils.java
@@ -113,7 +113,7 @@ public class KerasOptimizerUtils {
                 double beta1 = (double) optimizerParameters.get(BETA_1);
                 double beta2 = (double) optimizerParameters.get(BETA_2);
                 double epsilon = (double) optimizerParameters.get(EPSILON);
-                double scheduleDecay = (double) optimizerParameters.get(SCHEDULE_DECAY);
+                double scheduleDecay = (double) optimizerParameters.getOrDefault(SCHEDULE_DECAY,0.0);
 
                 dl4jOptimizer = new Nadam.Builder()
                         .beta1(beta1).beta2(beta2)

--- a/platform-tests/src/test/java/org/eclipse/deeplearning4j/frameworkimport/keras/configurations/KerasModelImportTest.java
+++ b/platform-tests/src/test/java/org/eclipse/deeplearning4j/frameworkimport/keras/configurations/KerasModelImportTest.java
@@ -55,6 +55,7 @@ class KerasModelImportTest extends BaseDL4JTest {
     public long getTimeoutMilliseconds() {
         return 9999999999999L;
     }
+    
 
     @Test
     @DisplayName("Test H 5 Without Tensorflow Scope")

